hydrolib/core/basemodel.py

```diff
@@ -607,14 +612,6 @@ def validate(cls: Type["FileModel"], value: Any):
             value = {"filepath": Path(value)}
         return super().validate(value)
 
-    def _load(self, filepath: Path) -> Dict:
-        # TODO Make this lazy in some cases
-        # so it doesn't become slow
-        if filepath.is_file():
-            return self._parse(filepath)
-        else:
-            raise ValueError(f"File: `{filepath}` not found, skipped parsing.")
-
     def save(self, filepath: Optional[Path] = None, recurse: bool = False) -> None:
```

Comment:
Should this not also be defined as `@abstractclassmethod`? Or why not?

Reply:
`@abstractclassmethod` indicates that the a subclass will need to define it. If any `@abstractclassmethod` or `@abstractmethod` is not implemented, initializing the subclass will raise an exception. 

In this case, the implementation of save is already defined here, and won't need to be defined in a subclass, which is why it is not abstract. It does rely on several methods which are abstract which need to be implemented. But, because we know that any concrete class has these implemented, `save` will work as expected.